ansible/roles/rhacm_manage_cluster/templates/hub.yml.j2

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    cluster.open-cluster-management.io/managedCluster: "{{managed_cluster_name}}"
+  name: "{{managed_cluster_name}}"
+---
+apiVersion: cluster.open-cluster-management.io/v1
+kind: ManagedCluster
+metadata:
+  name: "{{managed_cluster_name}}"
+spec:
```

Comment:
we might want to inject the label `vendor: OpenShift` into this so that observability addon is enabled by default